schema/traces/README.md

```diff
@@ -0,0 +1,104 @@
+`instrumentationScope.attributes.identification` is presented will be used to determine the resource origin of the signal and can be used to filter accordingly
+
+## Traces
+see [OTEL traces convention](https://github.com/open-telemetry/opentelemetry-specification/tree/main/semantic_conventions/trace)
+
+Traces in are defined implicitly by their Spans - In particular, a Trace can be thought of as a directed acyclic graph (DAG) of Spans, where the edges between Spans are defined as parent/child relationship.
+
+## Spans
+A span represents an operation within a transaction. Each Span encapsulates the following state:
+
+* An operation name
+* start and finish timestamp
+* Attributes list of key-value pairs.
+* Set of Events, each of which is itself a tuple (timestamp, name, Attributes)
+* Parent's Span identifier.
+* Links to causally-related Spans (via the SpanContext of those related Spans).
+* SpanContext information required to reference a Span. 
+
+### SpanContext
+Represents all the information that identifies Span in the Trace and is propagated to child Spans and across process boundaries.
+A **SpanContext** contains the tracing identifiers and the options that are propagated from parent to child Spans.
+
+* TraceId  - It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes. 
+* SpanId  -  It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+* Tracestate - carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
+
+Additional fields can be supported via the Attributes key/value store see [traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md)
+
+### Structure
+The default fields that are supported by the traces are
+- **TraceId**  : It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes.
+- **SpanId**   : It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+- **ParentId** : It is the identifier for a span's parent span.
+- **TraceState** : carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
```

Comment:
Missing link in 'For more details see this'?

Do you intend traceState to be sent unpacked? Otel just passes it around as a string.